games-util/pyfa/Manifest

```diff
@@ -1 +1,2 @@
-DIST pyfa-1.21.3.tar.gz 7793523 SHA256 4ac79598f2e605ba7d5cd29cfef7a94f426d4cd9e934d13cbb2071afd566de3c SHA512 b5a691873421ef2d3b01d8bfd7121c63f07b96c75837fd3b93a5b77b8163943336dcd5bb85f8dcc056ce61b5103d5331afd5a81c09eb06a7050112b5d1a26681 WHIRLPOOL 65d6c9a52f4076d9be620aa3ac2ec5da23aaff32c547925b34480f7fc82ef9334fa737374419f0bf1bd827b3c6e0f34306c0d5de1dc6d441e88e096fd8aadc13
+DIST pyfa-1.33.2.tar.gz 8596509 SHA256 69d4a284e888ea6b81fbb01931f74e2c6b54168b233a411d7b9d97498b253caf SHA512 333951ef53d8088c7264297402e072124a960feb1832eda2ea2dec5df2ddc438fd72ceb3f90c5fdad0fe3f9f2e32bb712db1d30bb3f8ded921d2b621b40270f2 WHIRLPOOL f9fcd3b2caebfe70f148415a0ad3011484ef65f464606f97c2b1b07323f215ae9bc873bb7ceb8b72f8f84cef040b589fcc96fadd2b7d2838740249dd6322c3dc
+DIST pyfa-1.34.0.tar.gz 8629709 SHA256 ab4bd1f5c574577b783fd43fc06df903ea26c5651d2a6ed2265966ed12f9997d SHA512 4642efd4de5be4add3c3f994d7be9dbeeb231fe86229f7944d52f265527c7a79698d3da5ccac3018e02c6000b0da888ad0e64382e12f74b7dd7b3398e7bb2d48 WHIRLPOOL 2bed440067f9545789791d8b41332e3bd48ec1d1b27beda7b48053d6c0bbe272e2069961643637d7adf0fb503fe3df8745ad8ff85071a173d3459ac24443e890
```

Comment:
Not a valid Manifest. We should see BLAKE2B and SHA512 only. Make sure your portage and repoman are at the latest stable.